File: src/01-easy/the-descent/solution.cs
using System;
class Solution {
    static void Main() {
        while (true) {
            int max = -1, idx = 0;
            for (int i = 0; i < 8; i++) {
                int h = int.Parse(Console.ReadLine());
                if (h > max) { max = h; idx = i; }
            }
            Console.WriteLine(idx);
        }
    }
}

File: src/01-easy/the-descent/solution.vb
Module Solution
    Sub Main()
        Do
            Dim max As Integer = -1
            Dim idx As Integer = 0
            For i As Integer = 0 To 7
                Dim h As Integer = CInt(Console.ReadLine())
                If h > max Then max = h : idx = i
            Next
            Console.WriteLine(idx)
        Loop
    End Sub
End Module
